org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -32,8 +32,6 @@ logging.addLevelName(SCLOG, "STATE")
 Reset = "\u001B[0m"
 Red = "\u001b[31m"
 Green = "\u001b[32m"
-Blue = "\u001B[34m"
```

Comment:
I don't agree with your changes to the colours here. What's wrong with blue and cyan for transitions and events? Also green and red was perfect for showing which states were active and inactive. 
The original colour scheme made the large amount of text output easier to process while also making it pleasing to look at. I think we should stick with what we have.

Reply:
Ok I will change it back

Reply:
On second thought, @Chestshire  @RuomeiYan what do you guys think?